drivers/SmartThings/matter-lock/profiles/base-lock.yml

```diff
@@ -4,6 +4,13 @@ components:
   capabilities:
   - id: lock
     version: 1
+    configuration:
```

Comment:
```suggestion
    config:
```

Reply:
Since you seem to have inside knowledge of this feature, any update on when it will be available in prod? I saw this comment on #298 and #290 as well.